src/lib/Components/Consignments/Screens/SelectFromPhotoLibrary.tsx

```diff
@@ -188,12 +202,7 @@ export default class SelectFromPhotoLibrary extends React.Component<Props, State
       <Theme>
         <ProvideScreenDimensions>
           <BottomAlignedButton onPress={this.doneTapped} buttonText="Done">
-            <ScrollView
-              style={{ flex: 1 }}
-              scrollsToTop={true}
-              onScroll={this.onScroll.bind(this)}
```

Comment:
I still haven't quite internalized the distinction between class functions and class properties that _are_ functions. So I see we changed `onScroll`:

```diff
-  onScroll(e) {
+  onScroll = (e: NativeSyntheticEvent<NativeScrollEvent>) => {
```

Is there a strong preference for one or the other, generally? Or does it depend on a case-by-case basis? Seems like not having to `.bind(this)` all the time would make `onScroll = ` preferred. I'd appreciate any clarity around nomenclature that I've gotten wrong, too 🙇

Reply:
> Seems like not having to .bind(this) all the time would make onScroll = preferred.

yep. Forgetting .bind is something i've seen cause subtle bugs occasionally.